### PR TITLE
Updating e2e test to remove annotations in GKE

### PR DESCRIPTION
**Pull request by chrislovecnm**
Often annotations are added to pods by other sources. When
testing with gke with network policies enabled the unit tests are
failing because annotations are added by Calico. I am removing
those annotations in this test.

I have some updates to the Makefile as well.

CLOSES ISSUE: https://github.com/cockroachdb/cockroach-operator/issues/45

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@chrisseto and @johnrk PTAL

File: Makefile [hunk 1]
```diff
@@ -27,9 +28,15 @@ test-short: generate manifests
 	$(TOOLS_WRAPPER) $(REGISTRY_PREFIX)/$(TEST_RUNNER_IMG) make native-test-short
 
 e2e-test:
-	$(TOOLS_WRAPPER) -v ${HOME}/.kube:/root/.kube -v ${HOME}/.config/gcloud:/root/.config/gcloud -e USE_EXISTING_CLUSTER=true $(REGISTRY_PREFIX)/$(TEST_RUNNER_IMG) go test -v ./e2e/...
+	#$(TOOLS_WRAPPER) -v ${HOME}/.kube:/root/.kube -v ${HOME}/.config/gcloud:/root/.config/gcloud -e USE_EXISTING_CLUSTER=true $(REGISTRY_PREFIX)/$(TEST_RUNNER_IMG) go test -v  -run TestCreatesSecureClusterWithGeneratedCert ./e2e/... 
+	$(TOOLS_WRAPPER) -v ${HOME}/.kube:/root/.kube -v ${HOME}/.config/gcloud:/root/.config/gcloud -e USE_EXISTING_CLUSTER=true $(REGISTRY_PREFIX)/$(TEST_RUNNER_IMG) go test -v  ./e2e/... > e2e-test-output.$(DATE_STAMP).log
```
Comment (chrisseto) [inline Makefile:32]: Maybe `tee` to the log file?

Comment (chrislovecnm) [reply Makefile:32]: Agreed.  I think we are running shell.  Let me work out the syntax or run bash so that the syntax is not insane.

Comment (chrislovecnm) [reply Makefile]: @chrisseto fixed. PTAL

**Pull request merged**